roles/dnsdist/README.rst

```diff
@@ -54,6 +54,11 @@ Name from the container in which Dnsdist will run.
 
 The host where Dnsdist will be reachable.
 
+.. zuul:rolevar:: dnsdist_hosts
+   :default: - "{{ dnsdist_host }}"
```

Comment:
Shouldn't we at least deprecate dnsdist_host then?

Reply:
Does not matter. If you have a single host, use dnsdist_host. Otherwise, overwrite everything with dnsdist_hosts. This way we are backwards compatible and don't have to change anything.